collections/schema.go

```
@@ -7,8 +7,8 @@ import (
 	"sort"
 	"strings"
 
-	"cosmossdk.io/core/appmodule"
-	"cosmossdk.io/core/store"
+	appmodule "cosmossdk.io/collections/corecompat"
```

golangci-lint (1.64.8)

ST1019: package "cosmossdk.io/collections/corecompat" is being imported more than once

(stylecheck)